<ui version="4.0" >
 <class>teamStats</class>
 <widget class="QWidget" name="teamStats" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>145</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>General Information</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QTextBrowser" name="generalInfo" >
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>237</red>
              <green>236</green>
              <blue>234</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>237</red>
              <green>236</green>
              <blue>234</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="memberList" >
     <column>
      <property name="text" >
       <string>Rank</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Score</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>WUs</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="printButton" >
     <property name="text" >
      <string>Print Team Stats</string>
     </property>
     <property name="icon" >
      <iconset resource="foldwatcher.qrc" >:/icons/icons/document-print.png</iconset>
     </property>
     <property name="autoExclusive" >
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle" >
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pdfButton" >
     <property name="text" >
      <string>Export Team Stats</string>
     </property>
     <property name="icon" >
      <iconset resource="foldwatcher.qrc" >:/icons/icons/application-pdf.png</iconset>
     </property>
     <property name="toolButtonStyle" >
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="foldwatcher.qrc" />
 </resources>
 <connections/>
</ui>
